src/ucs/sys/sys.c

```diff
@@ -917,6 +917,25 @@ void ucs_sys_free(void *ptr, size_t length)
     }
 }
 
+const char* ucs_sys_get_mem_lock_limit(char *str, size_t max_size)
+{
+    struct rlimit limit_info;
+
+    /* Return the value of the max locked memory which is set on the system
+     * (ulimit -l) */
+    if (getrlimit(RLIMIT_MEMLOCK, &limit_info) != 0) {
+        ucs_error("getrlimit() failed: %m. Unable to read system limits");
```

Comment:
ucs_warn

Reply:
actually, better remote this error print altogether

Reply:
why?

Reply:
because it's a basic syscall wrapper in UCS, the error should be reflected by return value
it's enough to report the limit is unknown and let the caller of this function to decide what to do